dockerspawner/dockerspawner.py

```diff
@@ -373,6 +373,20 @@ def get_container(self):
                 raise
         return container
 
+    def upgrade_kwags(self, data):
```

Comment:
This would perhaps be more descriptive as `apply_templates`

Reply:
seems like a good idea